testsuite/shared/src/main/java/org/jboss/as/test/shared/TestSuiteEnvironment.java

```diff
@@ -33,9 +33,24 @@ public static ModelControllerClient getModelControllerClient() {
      * @return the client
      */
     public static ModelControllerClient getModelControllerClient(final URI authConfigUri) {
+        return getModelControllerClient(authConfigUri, getServerAddress(), getServerPort());
+    }
+
+    /**
+     * Creates a client based on the {@linkplain #getServerAddress() server address} and
+     * {@linkplain #getServerPort() port}. If the {@code authConfigUri} is not {@code null} the it will be used to
+     * authenticate the client.
+     *
+     * @param authConfigUri the path too the authentication configuration or {@code null}
+     * @param address of the server
+     * @param port the management port of the server
+     *
+     * @return the client
+     */
+    public static ModelControllerClient getModelControllerClient(final URI authConfigUri, String address, int port) {
```

Comment:
Thanks for review, I agree with your comment however I would like to point that the method with original set of parameters is still available (line 35).